build/template-run-integration-tests.yaml

```diff
@@ -5,7 +5,7 @@ steps:
 
 - task: VSTest@2
   displayName: 'Run integration tests (.NET FX)'
-  condition: and(succeeded(), eq(variables['PipelineType'], ''))
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
```

Comment:
should this also be included for `OneBranch`?

Reply:
Yes, it is already included here
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c69edf145208e85ed3d64f3eb3b8ee44778d5c65/build/template-run-integration-tests.yaml#L23